study-builder/studies/atcp/snippets/question/text/text-question-password.conf

```diff
@@ -0,0 +1,55 @@
+{
+  include required("../../../../snippets/text-question.conf"),
+  "isRestricted": true,
+  "hideNumber": true,
+  "promptTemplate": {
+    "templateType": "HTML",
+    "templateText": "$prompt_pass",
+    "variables": [
+      {
+        "name": "prompt_pass",
+        "translations": [
+          {
+            "language": "en",
+            "text": "Password"
+          }
+        ]
+      }
+    ]
+  },
+  "placeholderTemplate": {
+    "templateType": "TEXT",
+    "templateText": "$pass_placeholder",
+    "variables": [
+      {
+        "name": "pass_placeholder",
+        "translations": [
+          {
+            "language": "en",
+            "text": "Password"
+          }
+        ]
+      }
+    ]
+  },
+  "validations": [
+    {
+      "ruleType": "REQUIRED",
+      "hintTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$pass_req_hint",
+        "variables": [
+          {
+            "name": "pass_req_hint",
+            "translations": [
+              {
+                "language": "en",
+                "text": "Your Password is required"
```

Comment:
Where is this snippet being used and what for? Collecting password using an activity seems a bit worrisome.